inc/classes/class_func.php

```diff
@@ -594,99 +644,127 @@ public function checkIP($ip)
             if (ereg("[^0-9]", $IPParts[$i])) {
                 return 0;
             }
+
             if ($IPParts[$i] > 255 or $IPParts[$i] < 0) {
                 return 0;
             }
         }
+
         return 1;
     }
 
-
-  #### Misc ####
-  // Types:  1 = Info, 2 = Warning, 3 = Error (be careful with 3)
+    /**
+     * @param string    $message
+     * @param int       $type       1 = Info, 2 = Warning, 3 = Error (be careful with 3)
+     * @param string    $sort_tag
+     * @param string    $target_id
+     * @return int
+     */
     public function log_event($message, $type = 1, $sort_tag = '', $target_id = '')
     {
         global $db, $auth;
 
         if ($message == '') {
             echo("Function log_event needs message defined! - Invalid arguments supplied!");
+
         } else {
             if ($sort_tag == '') {
                 $sort_tag = $_GET['mod'];
             }
-            $entry = $db->qry("INSERT INTO %prefix%log SET
-             userid = %int%,
-             description=%string%,
-             type=%string%,
-             sort_tag = %string%,
-             target_id = %int%,
-             script = %string%,
-             referer = %string%,
-             ip = INET6_ATON(%string%)
-             ", $auth['userid'], $message, $type, $sort_tag, $target_id, $_SERVER["REQUEST_URI"], $this->internal_referer, $_SERVER['REMOTE_ADDR']);
+
+            $entry = $db->qry("
+            INSERT INTO %prefix%log 
+            SET
+              userid = %int%,
+              description=%string%,
+              type=%string%,
+              sort_tag = %string%,
+              target_id = %int%,
+              script = %string%,
+              referer = %string%,
+              ip = INET6_ATON(%string%)", $auth['userid'], $message, $type, $sort_tag, $target_id, $_SERVER["REQUEST_URI"], $this->internal_referer, $_SERVER['REMOTE_ADDR']);
+
             if ($entry == 1) {
                 return 1;
             }
         }
-          return 0;
+
+        return 0;
     }
 
-  // Better use MasterSearch..
+    /**
+     * @param string    $current_page
+     * @param int       $max_entries_per_page
+     * @param int       $overall_entries
+     * @param string    $working_link
+     * @param string    $var_page_name
+     * @return array
+     */
     public function page_split($current_page, $max_entries_per_page, $overall_entries, $working_link, $var_page_name)
     {
         if ($max_entries_per_page > 0 and $overall_entries >= 0 and $working_link != "" and $var_page_name != "") {
-            if ($current_page == "") {
```

Comment:
Reason for removal?

Reply:
Here it will be checked if `$current_page` is empty. If empty, then the vars `$page_sql`, `$page_a` and `$page_b` will be set.
A couple of lines after `$current_page` will be checked again and has an `else` part.
In this `else` part the variables `$page_sql`, `$page_a` and `$page_b` will be overwritten.

See https://github.com/lansuite/lansuite/blob/master/inc/classes/class_func.php#L638-L651

As a result: This is dead code.

Reply:
Indeed, good find!